### PR TITLE
Draft proposition for the sqs messaging connector extension

### PR DESCRIPTION
**Pull request by ozangunalp**
Simple wrapper extension for sqs connector

### CODE REVIEW

File: sqs-messaging-connector/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </dependency>
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-reactive-messaging-aws-sqs</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.amazonservices</groupId>
+            <artifactId>quarkus-amazon-sqs</artifactId>
+        </dependency>
+        <dependency>
```
Comment (scrocquesel) [inline sqs-messaging-connector/runtime/pom.xml:28]: Is Netty mandatory, it should work with the AWS crt http implementation?

**Comment by scrocquesel** [comment]
Maybe add an integration test inspired from the  quick start.

File: bom/pom.xml [hunk 2]
```diff
@@ -324,6 +324,16 @@
                 <artifactId>quarkus-amazon-sqs-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.amazonservices</groupId>
+                <artifactId>quarkus-amazon-sqs-messaging-connector</artifactId>
```
Comment (scrocquesel) [inline bom/pom.xml:329]: Should we try to keep the existing convention at quarkus core ? Smallrye-reactive-messaging-aws-sqs ?

**Comment by Tincoquesse** [comment]
I have prepared a basic version of the integration test but cannot push my branch to your repository, which you fork from quarkus-amazon-services. How can I provide you with this code?

**Comment by scrocquesel** [comment]
I create a dedicated branch locally so we can contribute on it more easily

File: bom/pom.xml (hunk 2 above)
Comment (scrocquesel) [reply bom/pom.xml]: @ozangunalp Quarkus core extension ends with Connector ? If so, I would vote for `quarkus-messaging-amazon-sqs-connector` for artifactId and `Quarkus Messaging - Amazon Services - SQS Connector` as the extension name.
Seems to respect both Quarkus core naming convention and this repo ones

Comment (ozangunalp) [reply bom/pom.xml]: In Quarkus core the artifact id is  `quarkus-messaging-*`, artifact name is `Quarkus - Messaging - * - Runtime` etc. and the extension name is `Messaging * Connector`.

We can name this extension `Messaging - Amazon SQS - Connector` and the artifact `quarkus-messaging-amazon-sqs`

**Pull request closed without merging**